sdk/tables/azure-tables/review/tables.api.md

```diff
@@ -4,8 +4,595 @@
 
 ```ts
 
+import * as coreHttp from '@azure/core-http';
+
+// @public
+export interface AccessPolicy {
+    expiry: Date;
+    permission: string;
+    start: Date;
+}
+
+// @public
+export interface CorsRule {
+    allowedHeaders: string;
+    allowedMethods: string;
+    allowedOrigins: string;
+    exposedHeaders: string;
+    maxAgeInSeconds: number;
+}
+
+// @public
+export interface GeneratedClientOptionalParams extends coreHttp.ServiceClientOptions {
+    endpoint?: string;
+    version?: string;
+}
+
 // @public (undocumented)
-export function helloWorld(): string;
+export interface GeoReplication {
+    lastSyncTime: Date;
+    status: GeoReplicationStatusType;
+}
+
+// @public
+export type GeoReplicationStatusType = "live" | "bootstrap" | "unavailable";
+
+// @public
+export interface Logging {
+    delete: boolean;
+    read: boolean;
+    retentionPolicy: RetentionPolicy;
+    version: string;
+    write: boolean;
+}
+
+// @public (undocumented)
+export interface Metrics {
+    enabled: boolean;
+    includeAPIs?: boolean;
+    retentionPolicy?: RetentionPolicy;
+    version?: string;
+}
+
+// @public
+export type OdataMetadataFormat = "application/json;odata=nometadata" | "application/json;odata=minimalmetadata" | "application/json;odata=fullmetadata";
+
+// @public
+export interface QueryOptions {
+    filter?: string;
+    format?: OdataMetadataFormat;
+    select?: string;
+    top?: number;
+}
+
+// @public
+export type ResponseFormat = "return-no-content" | "return-content";
+
+// @public
+export interface RetentionPolicy {
+    days?: number;
+    enabled: boolean;
+}
+
+// @public
+export interface ServiceGetPropertiesHeaders {
+    // (undocumented)
+    xMsClientRequestId?: string;
+    // (undocumented)
+    xMsRequestId?: string;
+    // (undocumented)
+    xMsVersion?: string;
+}
+
+// @public
+export interface ServiceGetPropertiesOptionalParams extends coreHttp.OperationOptions {
+    requestId?: string;
+    timeout?: number;
+}
+
+// @public
+export type ServiceGetPropertiesResponse = ServiceGetPropertiesHeaders & TableServiceProperties & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: TableServiceProperties;
+        parsedHeaders: ServiceGetPropertiesHeaders;
+    };
+};
+
+// @public
+export interface ServiceGetStatisticsHeaders {
+    // (undocumented)
```

Comment:
We'll want to add documentation for these types (I think the most straight forward way to do that given that these are auto generated is to do transforms in the swagger README.md to add documentation, which the code generators will then include when they codegen.

Similar feedback applies for other undocumented types that we are including as part of our public API.

Reply:
It would be ideal if the descriptions are added in the swagger then we don't need swagger transformation.

Reply:
@mahdiva and I looked into this offline and we found that the Swagger does have the descriptions, however Autorest is not picking them up. There is a PR in @autorest/modelerfour to fix this Azure/autorest.modelerfour/pull/318

Reply:
Comments and Name Overrides for these models will be added by Autorest once the PR above mentioned is merged and the swagger is regenerated. Since this is an issue in Autorest, I'd suggest to check-in this way and send a follow up PR re-generating the client once the fixes in autorest are merged. @jeremymeng  @ellismg does that sound good?